src/networking.c

```diff
@@ -539,6 +539,116 @@ get_mac_address(const gchar *const ifname)
 	return macaddr;
 }
 
+/*
+ * Get and store the iptable rules set in the
+ * network namespace. These need to be re-applied
+ * within the container.
+ *
+ * \param config \ref cc_oci_config
+ *
+ * \return \c true on success, else \c false.
+ */
+static gboolean
+get_iptable_rules(struct cc_oci_config *config)
+{
+	gboolean ret;
+	GError *error = NULL;
+	gint exit_status = 0;
+	char *cmd = "iptables-save";
+	char *args[] = { cmd, NULL };
+
+	if (! config) {
+		return false;
+	}
+
+	g_debug("Running command : %s", cmd);
+	ret = g_spawn_sync (NULL,
+              args,
+              NULL,
+              G_SPAWN_SEARCH_PATH | G_SPAWN_STDERR_TO_DEV_NULL,
```

Comment:
I'd rather use full path 
according with glib doc
```
If the program name is not a full path and G_SPAWN_SEARCH_PATH flag is not used, then the program will be run from the current directory (or working_directory , if specified); this might be unexpected or even dangerous in some cases when the current directory is world-writable.
```